metacatalog/ext/export_standard/schemas/iso19115/iso19115-2.j2

```diff
@@ -0,0 +1,412 @@
+            </gmd:spatialRepresentationType>
+            <gmd:spatialResolution>
+                <gmd:MD_Resolution>
+                    {# is distance really resolution? 'Ground sample distance' #}
+                    <gmd:distance>
+                        <gco:Distance uom="meters">{{datasource.spatial_scale.resolution}}</gco:Distance>
+                    </gmd:distance>
+                </gmd:MD_Resolution>
+            </gmd:spatialResolution>
+            {% endif %}
+            {% endif %}
+
+            <gmd:language>
+                {# always english #}
+                <gco:CharacterString>eng</gco:CharacterString>
+            </gmd:language>
+            <gmd:characterSet>
+            {% if datasource %}
+                {# if datasource exists: use datasource.encoding #}
+                <gmd:MD_CharacterSetCode codeListValue="{{datasource.encoding|replace("-", "")}}" codeList="http://www.isotc211.org/2005/resources/codeList.xml#MD_CharacterSetCode"/>
+            {% else %}
+                {# no datasource: utf8 #}
+                <gmd:MD_CharacterSetCode codeListValue="utf8" codeList="http://www.isotc211.org/2005/resources/codeList.xml#MD_CharacterSetCode"/>
+            {% endif %}
+            </gmd:characterSet>
+
+
+            <gmd:topicCategory>
+                {# currently always 'environment', but other values are often better -> how do we map them? TODO #}
+                <gmd:MD_TopicCategoryCode>environment</gmd:MD_TopicCategoryCode>
+            </gmd:topicCategory>
+            
+            {% if datasource %}
+            <gmd:extent>
+                <gmd:EX_Extent id="boundingExtent">
+                    {# geographic extent only for datasources with spatial_scale #}
+                    {% if datasource.spatial_scale %}
+                    <gmd:geographicElement>
+                        <gmd:EX_GeographicBoundingBox id="boundingGeographicBoundingBox">
+                            {# TODO: get bounding box parameters from column extent (atm no example present) #}
+                            <gmd:westBoundLongitude>
+                                <gco:Decimal>{{datasource.spatial_scale.extent}}</gco:Decimal>
+                            </gmd:westBoundLongitude>
+                            <gmd:eastBoundLongitude>
+                                <gco:Decimal>{{datasource.spatial_scale.extent}}</gco:Decimal>
+                            </gmd:eastBoundLongitude>
+                            <gmd:southBoundLatitude>
+                                <gco:Decimal>{{datasource.spatial_scale.extent}}</gco:Decimal>
+                            </gmd:southBoundLatitude>
+                            <gmd:northBoundLatitude>
+                                <gco:Decimal>{{datasource.spatial_scale.extent}}</gco:Decimal>
```

Comment:
Is this correct? you always pass the same data into the west/south lon/lat element...